### ISSUE TITLE
[EPIC]: A11y Issues assigned to Chris

### ISSUE THREAD

**Issue by chrisgrimley** [IBMa/equal-access#2524]
### Hill

Assigned issues:

Remapping landmark rules to proper WCAG SC: https://github.com/IBMa/equal-access/issues/2242 [IN PROGRESS]
Treeview should have search capability for large dataset: https://github.com/carbon-design-system/carbon/issues/19474 [SOLUTION PROVIDED]

### Playback Demonstration

Issue report showing number of bugs assigned, resolved etc

### Metric

Number of issues closed per quarter
Dev/Designer satisfaction

### Comms

A11y team

### Project

able site

**Comment by chrisgrimley** [comment]
Closed ticket so that issues can be separately tracked in their correct place (rather than having a placeholder ticket for all tickets).
